jobconfig.toml

```diff
@@ -271,8 +273,8 @@
 [STATION_ENGINEER]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 2
-"Total Positions" = 3
+"Spawn Positions" = 6
+"Total Positions" = 6
```

Comment:
4-5 is enough imo. 5 is also the lowest our maps go in terms of starting slots

Reply:
Altered to 5 in both counts; Will see how this changes balance.